### DISCUSSION TITLE
Crosstalk Calibration

### DISCUSSION THREAD

**Discussion by PrashastiNama** [Broadcom/AFBR-S50-API#21] [Q&A]
Hey! I just started working on AFBR-S50 using F401RE board. I have successfully used and implemented the porting guide. Now I want to calibrate my sensor for crosstalk but I'm unable to do so. I am using this function
`Argus_XtalkCalibration_CLI(argus_hnd_t * hnd)`
before calling this function the handle to the sensor works fine but this is not taking proper value for the handle. What am I supposed to do?
Can someone help with this?
@c-berger  @FloVesuv @cycode-admin

**Comment by goetzfrv** [comment]
Hello PrashastiNama,
please clone or download the latest code from this Github repository. The xtalk guide is already implemented in the examples so you only need to enable the preprocessor setting from the example.h as described in https://docs.broadcom.com/docs/AFBR-S50-Cover-Glass-Calibration-AN.

**Reply by PrashastiNama** [reply]
I have downloaded the same code as you mentioned. It is not working. It says the handle is not initialized(as in the handle for sensor). But it is working till the previous statement. I don't understand what to do now.

**Reply by goetzfrv** [reply]
Can you please show the code to get a better understanding of where the error happens?
